### PR TITLE
Test mpOpenAPI config with checkpoint/restore

### PR DESCRIPTION
**Pull request by anjumfatima90**
for #26966

### CODE REVIEW

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = FATSuite.defaultMPRepeat(SERVER_NAME);
+
+    @BeforeClass
+    public static void deployApp() throws Exception {
+        // Set guards
+        server.setJvmOptions(Arrays.asList("-Dcom.ibm.ws.beta.edition=true"));
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java:69]: We will need to remember remove this JVM option.

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    private static String DEFAULT_DOC_PATH = "/openapi/docs";
+    private static String DEFAULT_UI_PATH = DEFAULT_DOC_PATH + "/ui";
+
+    private static String APP_NAME = "mpOpenAPIConfigApp";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = FATSuite.defaultMPRepeat(SERVER_NAME);
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java:64]: Can this really repeat.  I thought the settings tested here were specific to mpOpenApi-3.1

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Review by tjwatson — DISMISSED** [review]

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java [hunk 3]
```diff
@@ -89,7 +89,7 @@ public void setup() throws Exception {
     @After
     public void teardown() throws Exception {
         try {
-            server.stopServer();
+            server.stopServer("CWWKE0701E");
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPIConfigTest.java]: This is pretty general.  I was thinking you would use regex on the specific message:

```java
"CWWKE0701E: .*BundleContext is no longer valid io.openliberty.microprofile.openapi.2.0.internal.servlet"
```

**Comment by anjumfatima90** [comment]
Instant On build 
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_E3YVMJBgEe6gttl9kBC4bw&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider

**Review by tjwatson — APPROVED** [review]

**Comment by anjumfatima90** [comment]
#libby

**Pull request merged**